.travis.yml

```diff
@@ -1,11 +1,11 @@
 env:
   - REACT_APP_DISABLE_SOCKET=true
-  
+
 language: node_js
 git:
   depth: 1
 node_js:
-  - "12"
+  - "14"
```

Comment:
I'm used to installing `node` via `nvm` or `nodenv`. If we install `npm@7.6.3` in our Docker image does that also mean the system node is set to `14`?

Reply:
For our docker file we pull down the latest stable 14.x nodejs from NodeSource, it would come with npm 6 by default (npm 7 ships with nodejs 15).

For Travis we are still using npm 6, but I can also update it to use npm 7